src/editor/components/components/Sidebar.js

```diff
@@ -63,86 +72,126 @@ export default class Sidebar extends React.Component {
     this.setState({ showSideBar: !this.state.showSideBar });
   };
 
+  renderSidebarContent() {
+    const { entity } = this.props;
+
+    if (entity.id === 'reference-layers') {
+      return <GeoSidebar entity={entity} />;
+    }
+
+    if (entity.getAttribute('street-segment')) {
+      return (
+        <>
+          <StreetSegmentSidebar entity={entity} />
+          <div className="advancedComponentsContainer">
+            <AdvancedComponents entity={entity} />
+          </div>
+        </>
+      );
+    }
+
+    if (entity.getAttribute('managed-street')) {
+      return (
+        <>
+          <ManagedStreetSidebar entity={entity} />
+          <div className="advancedComponentsContainer">
+            <AdvancedComponents entity={entity} />
+          </div>
+        </>
+      );
+    }
+
+    const isIntersection = entity.getAttribute('intersection');
+    const hasNoTransform = entity.hasAttribute('data-no-transform');
+
+    return (
+      <>
+        {entity.mixinEls.length > 0 && <Mixins entity={entity} />}
+
+        {!hasNoTransform && (
+          <div id="sidebar-buttons">
+            <Button variant={'toolbtn'} onClick={() => cloneEntity(entity)}>
```

Comment:
i feel like we would want the ability to duplicate managedstreetsidebars no?

Reply:
yes we probably do, and also rename and delete!

Reply:
ok added these back for managed street 4f166e3226a84510f6e68a5a500a7ba6a7d91bbe